Fika.Core/Coop/Factories/PingFactory.cs

```diff
@@ -22,6 +23,24 @@ public enum EPingType : byte
         Interactable
     }
 
+    public static EUISoundType GetPingSound()
+    {
+        return FikaPlugin.PingSound.Value switch
```

Comment:
I suppose I could use a map of some sort for this, but I don't know C# well enough to be confident it's the right approach and this is the first Unity anything I've done.